src/pages/RepoPage/ComponentsTab/BackfillBanners/BackfillBanners.spec.jsx

```diff
@@ -0,0 +1,40 @@
+import { render, screen } from 'custom-testing-library'
+
+import BackfillBanners from './BackfillBanners'
+import { useRepoBackfillingStatus } from './hooks'
+
+jest.mock('./TriggerSyncBanner', () => () => 'TriggerSyncBanner')
+jest.mock('./SyncingBanner', () => () => 'SyncingBanner')
+
+jest.mock('./hooks')
+
+describe('BackfillBanner', () => {
+  function setup(data) {
+    useRepoBackfillingStatus.mockReturnValue(data)
```

Comment:
Yes this will be replaced by `useComponentsBackfilled` which calls the `BackfillComponentMemberships` query, will add a TODO here to replace with MSW when that switch is made in the coming step.